packages/x-components/src/__stubs__/related-tags-stubs.factory.ts

```diff
@@ -15,15 +15,6 @@ export function getRelatedTagsStub(amount = 12): RelatedTag[] {
   );
 }
 
-/**
- * Function to create selected related tags stub.
- *
- * @returns Array of selected related tags stub.
- */
-export function getSelectedRelatedTagsStub(): RelatedTag[] {
```

Comment:
We don't need this function anymore, there is no difference in the model of a selected related tag or an unselected one.